### PR TITLE
Cart 282 bloking towing notification

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/routing/blocking.js [hunk 1]
```diff
@@ -33,7 +35,9 @@ module.exports = function (server) {
       },
       handler: function (request, reply) {
         DAL.blocking.create(request.params.propertyId, request.payload, function (err, docs) {
-          !err ? reply(docs) : reply(JSON.stringify(err));
+          DAL.properties.getById(request.params.propertyId, (err, property) => {
+            !err ? reply(towingBlockingService.sendNotification(request.params.propertyId, property.name, request.auth.credentials.email)) : reply(Boom.badImplementation(err));
```
Comment (dmitrij-borchuk) [inline dev/routing/blocking.js]: Line is too long

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/routing/blocking.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/routing/blocking.js]: fixed

File: dev/routing/blocking.js [hunk 2]
```diff
@@ -32,8 +34,14 @@ module.exports = function (server) {
         }
       },
       handler: function (request, reply) {
-        DAL.blocking.create(request.params.propertyId, request.payload, function (err, docs) {
-          !err ? reply(docs) : reply(JSON.stringify(err));
+        DAL.blocking.create(request.params.propertyId, request.payload, 
+        function (err, docs) {
+          DAL.properties.getById(request.params.propertyId, (err, property) => {
+            !err ? reply(towingBlockingService.sendNotification(
```
Comment (dmitrij-borchuk) [inline dev/routing/blocking.js]: "towingBlockingService.sendNotification" returns promise. Promose should not be parameter of "reply"

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/routing/blocking.js (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/routing/blocking.js]: fixed

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**